doc/1/getting-started/java/index.md

```diff
@@ -0,0 +1,149 @@
+---
+code: false
+type: page
+title: Java
+description: Java Getting started
+order: 99
+---
+
+# Getting Started
+
+In this tutorial you will learn how to install the Kuzzle **Jvm SDK**.
+This page shows examples of scripts that **store** documents in Kuzzle, and of scripts that subcribe to real-time **notifications** for each new document created.
+
+::: success
+Before proceeding, please make sure your system meets the following requirements:
+
+- **Oracle JDK or OpenJDK** version 8 or higher ([OpenJDK installation instructions](https://openjdk.java.net/install/))
+- A running Kuzzle Server ([Kuzzle installation guide](/core/2/guides/essentials/installing-kuzzle))
+
+:::
+
+
+::: info
+Having trouble? Get in touch with us on [Discord](http://join.discord.kuzzle.io)!
+:::
+
+## Installation
+
+You can find the SDK JARs directly on [bintray](https://bintray.com/kuzzle/maven/sdk-jvm). Download and add it to your classpath.
+
+::: info
+The following examples are made to be executed without any IDE.
+If you're using Eclipse, IntelliJ or another Java IDE, you need to add the SDK as a project dependency in your classpath.
+:::
+
+### Installing on Android using gradle
+
+In your app build.gradle add the following line and sync.
+
+### Maven:
+
+```xml
+<dependency>
+  <groupId>io.kuzzle</groupId>
+  <artifactId>sdk-jvm</artifactId>
+  <version>1.0.0</version>
+  <type>pom</type>
+</dependency>
+```
+
+### Gradle:
+
+```groovy
+implementation 'io.kuzzle:sdk-jvm:1.0.0'
+```
+
+### Ivy:
+
+```html
+<dependency org='io.kuzzle' name='sdk-jvm' rev='1.0.0'>
+  <artifact name='sdk-jvm' ext='pom' ></artifact>
+</dependency>
+```
+
+## First connection
+
+Initialize a new Java project, create a `GettingStartedFirstConnection.java` file and start by adding the code below:
+
+<<< ./snippets/firstconnection.java
+
+This program initializes the Kuzzle Server storage by creating a index, and a collection inside it
+Run the program with the following command:
+
+```bash
+$ javac -classpath ./path/to/the/sdk.jar GettingStartedFirstConnection.java
+$ java -classpath .:./path/to/the/sdk.jar GettingStartedFirstConnection
+Connected!
+Index nyc-open-data created!
+Collection yellow-taxi created!
+```
+
+Congratulations, you performed your first connection to Kuzzle Server via a Java program.
```

Comment:
(open to debate) Should we really congratulate, especially over something as trifle as this?
I would favor something more formal, but that may be just me.

```suggestion
```

Reply:
I dont find it disturbing, I feel like I see this kind of thing in a lot of tutorials, but I dont have strong feelings about this.

Reply:
After thinking I remove it, maybe thats too much